lib/gateways/internal-api.ts

```diff
@@ -27,15 +27,24 @@ export const createApplication = async (application: Application) => {
     method: 'POST',
     body: JSON.stringify(application),
   });
-  return (await res.json()) as Application;
+
```

Comment:
Was this implemented by a previous PR? Guessing just out of date development at the time.